runner/internal/shim/api/schemas.go

```diff
@@ -8,6 +8,10 @@ type DockerTaskBody struct {
 	ImageName string `json:"image_name"`
 }
 
+type SubmitStopBody struct {
```

Comment:
There are still some `SubmitStop` in the code. But I guess it's fine

Reply:
Fixed.